doc/source/configuration/monitoring.rst

```diff
@@ -140,18 +140,34 @@ enable the ceph mgr exporter.
 OpenStack Capacity
 ==================
 
-OpenStack Capacity allows you to see how much space you have avaliable
-in your cloud. StackHPC Kayobe Config includes this exporter by default
-and it's necessary that some variables are set to allow deployment.
+OpenStack Capacity allows you to see how much space you have available
+in your cloud. StackHPC Kayobe Config includes a playbook for manual
+deployment, and it's necessary that some variables are set before
+running this playbook.
 
 To successfully deploy OpenStack Capacity, you are required to specify
 the OpenStack application credentials in ``kayobe/secrets.yml`` as:
 
 .. code-block:: yaml
 
-    secrets_os_exporter_auth_url: <some_auth_url>
-    secrets_os_exporter_credential_id: <some_credential_id>
-    secrets_os_exporter_credential_secret: <some_credential_secret>
+    secrets_os_capacity_auth_url: <some_auth_url>
+    secrets_os_capacity_credential_id: <some_credential_id>
+    secrets_os_capacity_credential_secret: <some_credential_secret>
+
+Additionally, you are required to enable a conditional flag to allow
+HAProxy and Prometheus configuration to be templated to controllers during deployment.
```

Comment:
In many environments, HAProxy and Prometheus are not deployed on controllers, so this could be confusing. You can just remove `to controllers`. Could you also explicitly state that both HAProxy and Prometheus must be reconfigured?